config/slack-random-response.json

```
@@ -74,7 +74,7 @@
   {
     "botName": "Lovin.gov Bot",
     "defaultEmoji": ":lovin-gov:",
-    "trigger": "need me some lovin.gov",
+    "trigger": "need me some (<http://lovin.gov\\|)?lovin.gov",
```

do we have to worry about it auto-adding https? ex:

```suggestion
    "trigger": "need me some (<https?://lovin.gov\\|)?lovin.gov",
```

---

I'm not sure, to be honest. But the `s?` is probably a good safety addition.